src/github.com/getlantern/flashlight/http/http.go

```diff
@@ -0,0 +1,167 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/getlantern/flashlight/client"
+	"github.com/getlantern/flashlight/config"
+	"github.com/getlantern/flashlight/util"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/whitelist"
+	"github.com/skratchdot/open-golang/open"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/getlantern/tarfs"
+)
+
+const (
+	UiUrl = "http://%s"
+	UiDir = "src/github.com/getlantern/ui/app"
```

Comment:
This assumes that flashlight is run from the root of flbuild. That's a little weird since flashlight is theoretically it's own project, so if someone isn't running it in the context of flbuild, this won't work.  I tend to run it from its own source folder (i.e. src/github.com/getlantern/flashlight).  Perhaps to clear up confusion, you could add a flag `localuidir` or something like that and give it a sensible default.